spec/log_parser_spec.rb

```diff
@@ -18,7 +18,10 @@
     let(:file) { LogParser.new('./spec/fixtures/game_test.log') }
 
     it 'returns JSON object' do
-      expect(file.parse_file).to include('{"./spec/fixtures/game_test.log":{"lines":5,"players":["Isgalamido","Dono da Bola","Mocinha')
+      expect(file.parse_file).to be == { './spec/fixtures/game_test.log' => {
+        'kills' => { 'Dono da Bola' => 2, 'Isgalamido' => 4,
+                     'Mocinha' => 1 }, 'lines' => 13, 'players' => ['Isgalamido', 'Dono da Bola', 'Mocinha'], 'total_kills' => 7
+      } }
```

Comment:
Depois que fazer voltar a retornar JSON, você pode fazer o assert dessa forma:

```suggestion
      parsed_log = {
        './spec/fixtures/game_test.log' => {
          'kills' => {
            'Dono da Bola' => 2,
            'Isgalamido' => 4,
            'Mocinha' => 1
          },
          'lines' => 13,
          'players' => ['Isgalamido', 'Dono da Bola', 'Mocinha'],
          'total_kills' => 7
        }
      }.as_json

      expect(file.parse_file).to eq(parsed_log)
```
Provavelmente precisa adicionar `require 'json'` no início do arquivo.